frontend/src/scenes/Auth/components/ForgotPassword/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC } from 'react';
+import { Formik } from 'formik';
+import { Button, Form } from 'react-bootstrap';
+import { IForgotPasswordInput } from '../../../../common/models/auth/IForgotPasswordInput';
+import { IBindingCallback1 } from '../../../../common/models/callback/IBindingCallback1';
+import forgotPasswordSchema from '../../../../common/models/formik/forgotPasswordSchema';
+import THandleForgotPassword from '../../../../common/models/auth/THandleForgotPassword';
+import initialValuesForgotPassword from '../../../../common/models/formik/initialValuesForgotPassword';
+
+import styles from './styles.module.scss';
+
+interface IProps {
+  forgotPassword: IBindingCallback1<IForgotPasswordInput>;
+}
+
+const ForgotPassword: FC<IProps> = ({ forgotPassword }) => {
+  const handleForgotPassword: THandleForgotPassword = async ({ email }, formikHelpers) => {
+    await forgotPassword({ email });
```

Comment:
no async functions in components, all async operations should be handled via sagas

Reply:
Solved